zmq/asyncio.py

```diff
@@ -49,23 +49,35 @@ class _AsyncIO(object):
     _Future = Future
     _WRITE = selectors.EVENT_WRITE
     _READ = selectors.EVENT_READ
-    
+
     def _default_loop(self):
         return asyncio.get_event_loop()
 
 
-class ZMQSelector(selectors.BaseSelector):
+class ZMQSelector(selectors._BaseSelectorImpl):
```

Comment:
It doesn't seem right to inherit from a private class, since we shouldn't assume it will continue to exist, even from one patch release to the next. Can this be done with public APIs?